src/Features/Core/Portable/Completion/Providers/AbstractInternalsVisibleToCompletionProvider.cs

```diff
@@ -0,0 +1,218 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.LanguageServices;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.Completion.Providers
+{
+    internal abstract class AbstractInternalsVisibleToCompletionProvider : CommonCompletionProvider
+    {
+        private const string ProjectGuidKey = nameof(ProjectGuidKey);
+
+        protected abstract IImmutableList<SyntaxNode> GetAssemblyScopedAttributeSyntaxNodesOfDocument(SyntaxNode documentRoot);
+        protected abstract SyntaxNode GetConstructorArgumentOfInternalsVisibleToAttribute(SyntaxNode internalsVisibleToAttribute);
+
+        internal override bool IsInsertionTrigger(SourceText text, int insertedCharacterPosition, OptionSet options)
+        {
+            var ch = text[insertedCharacterPosition];
+            return ch == '\"';
```

Comment:
@rchande 
See commit fa0bbd491faaf84abdd84b35cbd4ce5e8c23525e on how I implemented this. You earlier suggested [here](https://github.com/dotnet/roslyn/pull/19805#discussion_r124871060):

>Not sure if you and Cyrus discussed this, but seems like it would be reasonable to suggest names in quotes after the user types open paren. We do something similar for "cref" intellisense. Let me know if you want to try that.

Shall I still look into this or are we done for now.